src/bcf/report/oncoprint.rs

```diff
@@ -63,9 +63,9 @@ pub fn oncoprint(
     for (sample, path) in sample_calls.iter().sorted() {
         let bcf_reader = bcf::Reader::from_path(path)?;
         let header_records = bcf_reader.header().header_records();
-        let ann_fields: Vec<_> = get_ann_description(header_records).unwrap_or_else(|| {
-            panic!("No ANN field found. Please only use VEP-annotated VCF-files.")
-        });
+        let ann_fields: Vec<_> = get_ann_description(header_records)
+            .context("No annotations found. Please only use VEP-annotated VCF-files.")
+            .unwrap();
```

Comment:
Would be better to not unwrap here and below. Rather return the error.

Reply:
Changed the return type of `get_ann_description()` to `Result<T>`.